host_list.txt

```diff
@@ -1,3 +1,4 @@
+#test
 CODE1_AP_MYAPP1_LOCATION1_HOST1
 XY_AP_MYAPP2_LOCATION2_HOST2
 XY_MYAPP3_LOCATION3_HOST3
```

Comment:
actions:
      - "Check which API affected"
      - "Check ESB dashboard"
      - "Notify customer about the issue"